rollup.config.js

```diff
@@ -172,6 +173,9 @@ const ROLLUP = [
                     moduleDirectory: 'submodules/math-json/src',
                 },
             }),
+            polyfill([
+                '@webcomponents/webcomponentsjs/webcomponents-bundle.js',
```

Comment:
inject polyfill via rollup
polyfill injected in code doesn't work with multi-instance